src/features/offer/pages/tests/Offer.analytics.test.tsx

```diff
@@ -46,6 +46,19 @@ describe('<Offer /> - Analytics', () => {
     expect(analytics.logConsultWithdrawal).toHaveBeenCalledTimes(1)
   })
 
+  it('should trigger logOfferSeenDuration', async () => {
+    const offerPage = await renderOfferBodyPage()
+    expect(analytics.logOfferSeenDuration).not.toHaveBeenCalled()
+    expect(AppState.addEventListener).toHaveBeenCalled()
+    await act(async () => {
+      await offerPage.unmount()
```

Comment:
`unmount` n'est pas async, donc pas de mettre await non ?
```suggestion
      await offerPage.unmount()
```